feat(receiver-mock): mock processing time (#245)
SumoLogic/sumologic-kubernetes-tools: src/rust/receiver-mock/README.md

| `--port <port>`             | `-p <port>`       |     3000      | Port to listen on                                                                              |
| `--print-headers`           |                   |      N/A      | Use to print received request's headers                                                        |
| `--print-logs`              | `-r`              |      N/A      | Use to print received logs on stdout                                                           |
| `--print-metrics`           | `-m`              |      N/A      | Use to print received metrics (with dimensions) on stdout                                      |
| `--store-logs`              |                   |      N/A      | Use to store log data which can then be queried via `/logs/*` endpoints                        |
| `--store-metrics`           |                   |      N/A      | Use to store metrics which can then be returned via `/metrics-samples` endpoint                |
| `--version`                 | `-V`              |      N/A      | Print version information                                                                      |

> | `--delay-time` <delay_time> | `-t <delay_time>` |       0       | Use to specify delay time. It mocks request processing time in milliseconds.                   |

Reviewer: ```suggestion
| `--delay-time` <delay_time> | `-t <delay_time>` |       0       | Use to specify processing delay in milliseconds which will be added to every handled request.      |
```
